reco_utils/dataset/pandas_df_utils.py

```diff
@@ -315,3 +320,51 @@ def negative_feedback_sampler(
     )
 
     return df_sample
+
+
+def has_columns(df, columns):
+    """Check if DataFrame has necessary columns
+
+    Args:
+        df (pd.DataFrame): DataFrame
+        columns (list(str): columns to check for
+    Returns:
```

Comment:
minor comment, when I was using sphinx for another project, I realized that if we don't have a new line between the end of Args and Returns, sphinx won't parse the documentation correctly. I had to change all my code adding the new line, which was very annoying

Reply:
ah, that is annoying. ok I'll put the spaces back. I wish black fixed that.